pkg/server/commands/check_command.go

```diff
@@ -122,9 +150,18 @@ func (c *CheckQuery) Execute(ctx context.Context, params *CheckCommandParams) (*
 
 		return nil, nil, err
 	}
+
+	var datastoreQueryCount uint32
 	if resp != nil {
-		resp.ResolutionMetadata.DatastoreQueryCount = c.datastore.GetMetrics().DatastoreQueryCount
+		datastoreQueryCount = c.datastore.GetMetrics().DatastoreQueryCount
+		resp.ResolutionMetadata.DatastoreQueryCount = datastoreQueryCount
 	}
+
+	checkDurationHistogram.WithLabelValues(
+		utils.Bucketize(uint(datastoreQueryCount), []uint{50, 200}),
```

Comment:
`{50, 200}` are the same values [from our default config](https://github.com/openfga/openfga/blob/main/internal/server/config/config.go#L577). I'm not sure where exactly to put the bucket definitions in this context. The default config values are attached to the `server` object and not available here, and passing a measurement concern through to the commands feels like muddying the waters.

Any ideas?

Reply:
I don't like hardcoded values, so i prefer passing down `requestDurationByQueryHistogramBuckets` from the server to the command.

Reply:
Josh and I had a conversation about this work and decided to change the approach entirely. Summarizing:

Instead of having these metrics emitted by the `check_command` itself, we're going to add a `Duration` to our existing [ResolveCheckResponseMetadata](https://github.com/openfga/openfga/blob/main/internal/graph/resolve_check_response.go#L3). The calling server methods will then be responsible for using (or not using) that information to emit metrics as they see fit. This way we don't need to pass in a `caller`, we don't need to worry about passing through bucket configuration, and we have a much cleaner separation of concerns.

Am going to refactor starting right now